netlify.toml

```diff
@@ -1,7 +1,7 @@
 [build]
   base = "site/"
   command = "hugo --gc --minify --baseURL $BASE_URL"
-  publish = "site/public"
+  publish = "public"
```

Comment:
Looks like the deploy preview failed:

`Failed during stage 'building site': Deploy directory 'public' does not exist` 

Is that due to this line perhaps?

Reply:
Yup, "worked on my machine" but apparently that's not good enough for Netlify :)
Pushed the change, now the short links work: https://deploy-preview-2895--velero.netlify.app/docs/install-overview